src/time_tracker/timers/pubsub/commands.clj

```diff
@@ -13,35 +14,33 @@
 ;; Every message pushed from the server should have a :type field, and other args
 ;; as necessary.
 
-(defn stop-timer!
-  [channel google-id connection {:keys [timer-id stop-time] :as args}]
+(defn stop-timer-now!
+  [channel google-id connection {:keys [timer-id] :as args}]
   (if-let [stopped-timer (timers-db/stop!
-                          connection timer-id stop-time)]
+                          connection timer-id (util/current-epoch-seconds))]
     (io/broadcast-state-change! google-id stopped-timer :update)
     (io/send-error! channel "Could not stop timer")))
 
-(defn start-timer!
-  [channel google-id connection {:keys [timer-id started-time] :as args}]
+(defn start-timer-now!
+  [channel google-id connection {:keys [timer-id] :as args}]
   (if-let [{:keys [started-time duration] :as started-timer}
-           (timers-db/start! connection timer-id started-time)]
+           (timers-db/start! connection timer-id (util/current-epoch-seconds))]
     (do (io/broadcast-state-change! google-id started-timer :update)
         (let [started-timers (timers-db/retrieve-started-timers connection
                                                                 google-id)
               timers-to-stop (filter #(not= (:id %) timer-id)
                                      started-timers)]
           (doseq [timer timers-to-stop]
-            (stop-timer! channel google-id connection
-                         {:timer-id  (:id timer)
-                          :stop-time started-time}))))
+            (stop-timer-now! channel google-id connection
+                             {:timer-id  (:id timer)}))))
     (io/send-error! channel "Could not start timer")))
 
-(defn create-and-start-timer!
-  [channel google-id connection {:keys [project-id started-time created-time notes] :as args}]
+(defn create-and-start-timer-now!
```

Comment:
In a later iteration, we should separate the commands to create a timer and start a timer, so that you can create a timer without starting it.